src/vs/editor/contrib/codeAction/browser/codeActionWidgetContribution.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { editorConfigurationBaseNode } from 'vs/editor/common/config/editorConfigurationSchema';
+import * as nls from 'vs/nls';
+import { ConfigurationScope, Extensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+
+Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfiguration({
+	...editorConfigurationBaseNode,
+	properties: {
+		'editor.experimental.codeActionWidget.enabled': {
+			type: 'boolean',
+			scope: ConfigurationScope.LANGUAGE_OVERRIDABLE,
+			description: nls.localize('codeActionWidget', "Enable/disable opening the experimental Code Action Widget."),
+			default: false,
```

Comment:
Also make sure to add `tags: ['experimental']` (https://github.com/microsoft/vscode/blob/50056f3e78836797a68f6e8a9792c8b415f23ebe/src/vs/workbench/browser/workbench.contribution.ts#L110)